packages/office-ui-fabric-react/src/components/ScrollablePane/ScrollablePane.doc.tsx

```diff
@@ -29,6 +32,13 @@ export const ScrollablePanePageProps: IDocPageProps = {
       codepenJS: ScrollablePaneDetailsListExampleCodepen,
       view: <ScrollablePaneDetailsListExample />,
       isScrollable: false
+    },
+    {
+      title: 'Details List Locked Header & Footer (Optimized for Performance)',
```

Comment:
```suggestion
      title: 'Details List Locked Header & Footer (optimized for scroll)',
```

Reply:
It is actually optimized for perf and not specifically for scroll